distribution/docs/src/main/jdocs/content/_configuring/configuring-solr-appender-contents.adoc

```diff
@@ -0,0 +1,43 @@
+:title: Configuring Solr Appender
+:type: configuringAdminConsole
+:status: published
+:summary: Configuring Solr appender.
+:order: 19
+
+==== Solr Appender
+
+The Solr Appender is a Decanter collector which persists to Solr all events posted to `decanter/collect/*`.
+The batch size and and flush interval for the appender can be configured.
+
+===== Configure Solr Appender
+
+. Navigate to the *${admin-console}*.
+. Select the *${ddf-platform}* application.
+. Select the *Configuration* tab.
+. Select the *Solr Appender*.
+.. A dialogue will pop up that allows you to edit the Solr appender properties.
```

Comment:
✏️  Similar to my previous comment, consider rewording to avoid using "you" here.